repo2docker/app.py

```diff
@@ -243,14 +247,20 @@ def initialize(self):
             # remove the additional newline from the stream handler
             self.log.handlers[0].terminator = ''
             # We don't want a [Repo2Docker] on all messages
-            self.log.handlers[0].formatter = logging.Formatter(fmt='%(message)s')
+            self.log.handlers[0].formatter = logging.Formatter(
+                fmt='%(message)s'
+                )
 
         if args.image_name:
             self.output_image_spec = args.image_name
         else:
             # Attempt to set a sane default!
             # HACK: Provide something more descriptive?
-            self.output_image_spec = 'r2d' + escapism.escape(self.repo, escape_char='-').lower() + str(int(time.time()))
+            self.output_image_spec = (
+                'r2d' +
+                escapism.escape(self.repo, escape_char='-').lower() +
+                str(int(time.time()))
+                )
```

Comment:
Here too - looks like all closing parens are in line with args, rather than flush